tensorboard/plugins/pr_curve/summary.py

```diff
@@ -31,24 +31,19 @@ def op(
     labels,
     predictions,
     num_thresholds=200,
-    weights=None,
-    num_classes=1,
+    weight=1.0,
     display_name=None,
-    description=None):
-  """Computes multi-class PR summaries for a list of thresholds in `[0, 1]`.
+    description=''):
```

Comment:
Actually, I'd recommend against this change: it's TensorFlow convention to take optional arguments as `None` and, if necessary, set `if foo is None: foo = default_foo` within the code. This has the distinct advantage that it is always safe to pass `foo=None` to mean "use the default value," as opposed to having

```
def function(foo="DEFAULT"):
  pass

def caller(foo="DEFAULT"):
   function(foo=foo)

def grandcaller(foo="DEFAULT"):
  caller(foo=foo)
```
which both leaks implementation details of the default value and leaves the code much more fragile to changes.

(There are other benefits to `foo=None`, like consistent mutable-default-value initialization.)